Sources/MapboxMaps/Foundation/Extensions/Core/MapSnapshotOptions.swift

```diff
@@ -1,6 +1,18 @@
-import Foundation
+/// Set of options for taking map snapshot with Snapshotter.
+public struct MapSnapshotOptions {
+    /// Dimensions of the snapshot in points.
+    public let size: CGSize
+
+    /// Ratio between the number device-independent and screen pixels.
+    public let pixelRatio: CGFloat
+
+    /// Glyphs rasterization options to use for client-side text rendering.
+    /// By default, `GlyphsRasterizationOptions` will use `.ideographsRasterizedLocally`
+    public let glyphsRasterizationOptions: GlyphsRasterizationOptions?
```

Comment:
Could we set the default here?

Reply:
Yep; we might want to keep the type as optional though matching the underlying type. Adding a default would match what we had prior to this PR.